patterns/testimonials-single.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Testimonials Single
+ * Slug: woocommerce-blocks/testimonials-single
+ * Categories: WooCommerce
+ */
+?>
+<!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column {"width":"160px"} -->
+	<div class="wp-block-column" style="flex-basis:160px"><!-- wp:image {"align":"left","width":164,"height":164,"sizeSlug":"large","style":{"border":{"radius":"100%"}},"className":"is-style-rounded"} -->
+		<figure class="wp-block-image alignleft size-large is-resized has-custom-border is-style-rounded"><img src="https://s.w.org/images/core/5.8/portrait.jpg" alt="" style="border-radius:100%" width="164" height="164"/></figure>
+		<!-- /wp:image --></div>
+		<!-- /wp:column -->
+
+		<!-- wp:column {"layout":{"type":"default"}} -->
+		<div class="wp-block-column"><!-- wp:heading {"level":4,"style":{"typography":{"textTransform":"none","fontSize":"18px"}}} -->
+		<h4 class="wp-block-heading" style="font-size:18px;text-transform:none"><strong>Great experience</strong></h4>
+		<!-- /wp:heading -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"18px"}}} -->
+		<p style="font-size:18px">
+					In the end the couch wasn't exactly what I was looking for but my experience with the
```

Comment:
When inserted in the editor, it looks like there's an extra space at the beginning
<img width="676" alt="Screenshot 2023-04-25 at 10 58 21" src="https://user-images.githubusercontent.com/186112/234227403-ff25c9d6-fea4-4ebd-8549-6f8003468b22.png">